Added functionality to change font size, and ability to save/load settings.

IL-faith opened the pull request:
Notes:
1. All forms have access to above functionality using static methods. 
2. When to save/change settings (such as user checking a different radio button for font size) still needs to be implemented. 
3. Settings does not include color, only font size.

IL-faith committed:
Added functionality to change font size, and ability to save/load settings.
All forms have access to above functionality using static methods.
When to save/change settings (such as user checking a different radio button for font size) still needs to be implemented.
Settings does not include color, only font size.

laurenesco commented:
Can you add a button on the Discovery form where the font size selection will be so that I can test this?

IL-faith committed:
added radio buttons to Discovery Form for testing.
Note: placement of buttons isn't necessarily where final placement of buttons will be, nor stylized correctly.

IL-faith commented:
> Can you add a button on the Discovery form where the font size selection will be so that I can test this?

Just did it. Sorry it's late.

eliseomrico commented:
Faith stated that this PR was for testing. Closing PR.
